chapters/en/slides/module3/module3_03.Rmd

```diff
@@ -0,0 +1,245 @@
+
+Notes:
+For this exercise we will use another movies dataset containing more genres.
+The question we will try to answer is
+"Which genres has the highest worldwide gross?".
+
+---
+
+## Many distributions can't be effectively compared with histograms
+
+```{python}
+alt.Chart(movies_extended).mark_bar().encode(
+    alt.X('Worldwide Gross', bin=alt.Bin(maxbins=30)),
+    alt.Y('count()'),
+    alt.Color('Major Genre'))
+```
+
+Notes:
+This visualization is not effective at all
+in answering our question.
+Most genres have a really low worldwide gross,
+and because our histogram is stacked
+this bar dominates with around 2000 movies,
+and extends the y-axis so that it is hard to see 
+how many observations there are for highest
+x-values
+(but we know there are some since the axis extends that far).
+
+---
+
+## Many distributions can't be effectively compared with densities either
+
+```{python}
+(alt.Chart(movies_extended).mark_area().transform_density(
+    'Worldwide Gross',
+    groupby=['Major Genre'],
+    as_=['Worldwide Gross', 'density'])
+ .encode(
+    alt.X('Worldwide Gross'),
+    alt.Y('density:Q'),
+    alt.Color('Major Genre')))
+```
+
+Notes:
+Although we saw previously that a layered density chart
+is better than a stacked histogram,
+it is no effective in comparing this many distributions
+and has many of the same problems as the histogram in the previous slide.
+
+So what can we do to create an effective comparison between all the genres?
+
+---
+
+## Bar charts are effective for comparing a single value but hides any variation
+
+```{python}
+alt.Chart(movies_extended).mark_bar().encode(
+    alt.X('mean(Worldwide Gross)'),
+    alt.Y('Major Genre'))
+```
+
+Notes:
+We could use a barplot if we were only interested in comparing a single value
+from each distribution,
+such as the mean or median.
+However,
+this is generally not a good idea because any plot that only shows a single value
+will hide the variation in our data
+and could lead to us drawing the wrong conclusion
+as you will see in the next slide.
+
+---
+
+## Showing a single value can lead to incorrect conclusions
+
+<img src=/module3/barplot-hiding-points.png></img>
+
+[Beyond Bar and Line Graphs: Time for a New Data Presentation Paradigm](https://journals.plos.org/plosbiology/article?id=10.1371/journal.pbio.1002128)
+
+Notes:
+This image illustrates how the same barplot (A)
+could have been generated from several different distributions (B, C, D, or E).
+If we just looked at the barplot,
+we could not tell which of the distributions it came from,
+but looking at the points directly
+we would arrive at widely different conclusion to what was going on with our data.
+
+This is why we most of the times need to present a richer representation of the data
+such as a histogram or a density plot.
+Since these do not work well for multiple distributions,
+we will use a compromise: the boxplot.
+
+---
+
+## Boxplots show several key statistic from a distribution
```

Comment:
It would be nice to recreate this graphic, showing raw data points (jitter), a density and/or histogram and boxplot all lined vertically on top of one another. This would more clearly illustrate what the boxplot is showing. Also, let's label the box plot with percentages as well? Or even just percentiales? They are easier to understand than quartile, in particular, they will have no idea what an interquartile range is.

Reply:
Yeah, Hayley also pointed out that we need to explain IQR etc more. I will make a simplified versus of this when revising instead of the one I copied in when making the draft.

Reply:
Tracked in #102